### DISCUSSION TITLE
能否参考该软件

### DISCUSSION THREAD

**Discussion by badspider7** [Planshit/Tai#349] [开发者闲聊]
你好，我想问下可以参考这个软件的ui和思路，使用electron复现吗，只是学习用途，想着练练手。
这个软件很棒，ui也很不错。

**Comment by White-NX** [comment]
Tai使用MIT开源协议，任何人都有对它进行使用、复制、修改、合并 、发布、分发、再许可的权利。

有关开源协议的内容可以参考[Wikipedia上关于开源协议的页面](https://zh.wikipedia.org/wiki/%E5%BC%80%E6%BA%90%E5%8D%8F%E8%AE%AE)
